kyber_test.go

```diff
@@ -263,11 +267,11 @@ func GroupTest(t *testing.T, g kyber.Group) {
 }
 
 func TestKyberG1(t *testing.T) {
-	GroupTest(t, NewGroupG1())
+	GroupTest(t, NewGroupG1(nil))
```

Comment:
could consider a separate `NewGroupG1WithoutDST` func so as not to surprise any callers, and just panic if `dst` is `nil` in the `NewGroupG1` func

Reply:
💯

Reply:
I've just sent a new set of changes using variadic `byte` args to not break the existing public APIs, let me know if you prefer to have 2 functions.